airbyte-integrations/connectors/destination-tally/destination_tally/utils.py

```diff
@@ -369,15 +372,18 @@ def insert_sales_order_to_tally(config: Mapping[str, Any], data: Dict[str, Any],
     sales_order_payload = prepare_sales_order_payload(data=data, logger=logger)
 
     try:
+        logger.info(f"sales order payload : {sales_order_payload}")
         response = requests.request(method="POST", url=sales_order_template_url, data=sales_order_payload, headers=sales_order_headers)
+        if (response.status_code == 200) and ("processed successfully" in str(response.content).lower()):
+            logger.info(
+                f'sales order [Customer name = {data["Customer Name"]} , Voucher number = {data["Voucher Number"]}] successfully inserted into Tally'
+            )
+        else:
+            logger.warn(
+                f'sales order [Customer name = {data["Customer Name"]} , Voucher number = {data["Voucher Number"]}] cannot be inserted into Tally, Error : {response.content}'
+            )
     except Exception as e:
         logger.error(f"request for sales order not successful, {e}")
```

Comment:
let us put logger.exception everywhere where the exception is excepted so that we can see the stack trace